curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What command is used to see the path of working directory?
 
 #### --distractors--
 
-Placeholder distractor 1
+`mv`
 
 ---
 
-Placeholder distractor 2
+`cd`
 
 ---
 
-Placeholder distractor 3
+`ls`
 
 #### --answer--
 
-Placeholder answer
+`pwd`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to see what is present in a file?
 
 #### --distractors--
 
-Placeholder distractor 1
+`touch`
 
 ---
 
-Placeholder distractor 2
+`mv`
 
 ---
 
-Placeholder distractor 3
+`ls`
 
 #### --answer--
 
-Placeholder answer
+`more`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to create a new file?
 
 #### --distractors--
 
-Placeholder distractor 1
+`mkdir`
 
 ---
 
-Placeholder distractor 2
+`cd`
 
 ---
 
-Placeholder distractor 3
+`ls`
 
 #### --answer--
 
-Placeholder answer
+`touch`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to remove a file?
 
 #### --distractors--
 
-Placeholder distractor 1
+`cp`
 
 ---
 
-Placeholder distractor 2
+`cd`
 
 ---
 
-Placeholder distractor 3
+`mv`
 
 #### --answer--
 
-Placeholder answer
+`rm`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to copy a particular file to a folder?
 
 #### --distractors--
 
-Placeholder distractor 1
+`rm <file> <destination>`
 
 ---
 
-Placeholder distractor 2
+`mv <file> <destination>`
 
 ---
 
-Placeholder distractor 3
+`cd <file> <destination>`
 
 #### --answer--
 
-Placeholder answer
+`cp <file> <destination>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to see the list of files an folder present in a directory?
 
 #### --distractors--
 
-Placeholder distractor 1
+`touch`
 
 ---
 
-Placeholder distractor 2
+`ls -l`
 
 ---
 
-Placeholder distractor 3
+`mv`
 
 #### --answer--
 
-Placeholder answer
+`ls`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which will be used to rename the 'menlo.font' to 'menlo.otf'
 
 #### --distractors--
 
-Placeholder distractor 1
+`rm menlo.font menlo.otf`
 
 ---
 
-Placeholder distractor 2
+`cp menlo.font menlo.otf`
 
 ---
 
-Placeholder distractor 3
+`ls menlo.font menlo.otf`
 
 #### --answer--
 
-Placeholder answer
+`mv menlo.font menlo.otf`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command you will be used to find things or view a file tree ?
```

Comment:
```suggestion
Which command can be used to find files or view a file tree ?
```

Reply:
I have tried to resolve a question issue and grammar fixes. If there is any inconsistency let me know again